### PR TITLE
DOC-1935: TinyMCE 6.4.2 docs release `template`

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1935: 6.4.2 Enterprise Docs release.

Changes:
* TinyMCE 6.4.2 docs release `template` for `staging`

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] ~Documentation Team Lead has reviewed~

### CODE REVIEW

**Comment by kemister85** [comment]
Note: Once `6.4.2` is released, update the .api-version to `6.4.2` before merging.

**Comment by kemister85** [comment]
This PR is for creating a `template` placeholder for 6.4.2 release-notes.  All relevant tickets that are to be documented in the release-notes specific will be `merged` under individual PRs for approval by the assignee `developer`.

**Review by mkzhx — APPROVED** [review]

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -398,6 +398,16 @@
 ** xref:tinymce-and-cors.adoc[Cross-Origin Resource Sharing (CORS)]
 * Release information
 ** xref:release-notes.adoc[Release notes for TinyMCE 6]
+// *** TinyMCE 6.4.2
```
Comment (mkzhx) [inline modules/ROOT/nav.adoc:401]: I assume these are supposed to be commented out

Comment (kemister85) [reply modules/ROOT/nav.adoc:401]: Yes, as they are just a placeholder for now.

**Review by MitchCTiny — APPROVED** [review]

**Pull request merged**